src/components/HOCs/WithSortedChallenges/WithSortedChallenges.test.js

```diff
@@ -1,7 +1,7 @@
 import { sortChallenges } from './WithSortedChallenges'
 
 let basicProps = null
-let WrappedComponent = null
+// let WrappedComponent = null
```

Comment:
This is another example where the line wasn't really adding any value and I think it's okay to just delete it rather than preserving it in a comment.